CHANGELOG.md

```diff
@@ -1,6 +1,8 @@
 # Changelog
 
-## v0.7.0 - TBD
+## v1.0.0 - 2021-09
```

Comment:
s/2021-09/TBD/?

Reply:
I am planning to release this in the next PR.